Aws/Ec2/Instance.py

```diff
@@ -0,0 +1,16 @@
+from Aws.BaseObject import BaseObject
+
+
+class Instance(BaseObject):
+    """
+    EC2 Region
+    """
+    def __init__(self, instance_ids):
+        """
+        Initialize instance object
+
+        :param instance_ids: The ID's of the Ec2 instances
+        :type instance_ids: list
+        """
+        # Call parent object init function
+        super().__init__(instance_ids)
```

Comment:
this class is a child of BaseObject, you are calling BaseObject.__init__ function here- it needs the ARN, you can't just change this to an instance ID